src/Microdown-DocumentBrowser/MicDocumentBrowser.class.st

```diff
@@ -454,32 +407,19 @@ MicDocumentBrowser >> setSourceShowing: aBoolean [
 
 { #category : #layout }
 MicDocumentBrowser >> setTopLayout [
-
-	self layout: (self slideMode 
-		ifTrue:[ self setSlideShowLayout]
-		ifFalse: [ self setBrowserLayout ] )
-]
-
-{ #category : #slides }
-MicDocumentBrowser >> slideMode [
-	^ slideMode state
-]
-
-{ #category : #slides }
-MicDocumentBrowser >> slideToggleButton [
-	slideMode ifNil: [ 
-		slideMode := SpCheckBoxPresenter new.
-		slideMode
-			label: 'Slide mode';
-			state: false; 
-			whenChangedDo: [ :val | slideMode state: val. self setTopLayout  ];
-			yourself].
-	^ slideMode
+	self layout: ( SpBoxLayout newTopToBottom
+		add: (SpPanedLayout newLeftToRight
+					add: spDocumentList;
+					add: self setDocumentlayout ;
+					positionOfSlider: 0.3;
+					yourself);
+		add: spButtonBar expand: false;
+		yourself)
 ]
 
 { #category : #'source code' }
 MicDocumentBrowser >> sourceMode [
-	^ sourceMode ifNil: [ sourceMode := #Horizontal ]
+	^ sourceMode ifNil: [ sourceMode := 'Source bottom' ]
```

Comment:
In fact I would define 4 methods sourceBottom ^ 'Source bottom' and I would use these methods to compare and more.